timemachine/ff/handlers/nonbonded.py

```diff
@@ -259,7 +259,7 @@ def compute_or_load_bond_smirks_matches(mol, smirks_list):
     return np.array(bond_idxs), np.array(type_idxs)
 
 
-def apply_bond_charge_corrections(initial_charges, bond_idxs, deltas):
+def apply_bond_charge_corrections(initial_charges, bond_idxs, deltas, runtime_validate=True):
```

Comment:
Nice -- this corrects an oversight on my part from https://github.com/proteneer/timemachine/pull/571